cpp/autosar/test/rules/A2-3-1/test.cpp

```diff
@@ -4,9 +4,15 @@
 double α = 2.;                   // NON_COMPLIANT; U+03b1
 void *to_𐆅_and_beyond = nullptr; // NON_COMPLIANT; U+10185
 int l1_\u00A8;                   // COMPLIANT[FALSE_POSITIVE]
-const char *euro = "α";          // NON_COMPLIANT
+const char *euro1 = "α";         // NON_COMPLIANT
+const wchar_t *euro2 = L"α";     // COMPLIANT
+const char *euro3 = u8"α";       // COMPLIANT
 
 int valid;
 /*
 Invalid character ↦ NON_COMPLIANT
+*/
+
+/*
+Valid character @ in comments COMPLIANT
```

Comment:
oh wait, what was this addition meant to be? if it was going to address the other exception case, add that , or else rm this

Reply:
This is an exception that was not part of our tests yet. Therefore I added it.

Reply:
aha yep! understood now!